### DISCUSSION TITLE
Please give some guestimate on a wait time

### DISCUSSION THREAD

**Discussion by yarikoptic** [community/community#25862] [Actions]
<p>For a new project I would like to start CI with github actions.  But currently from <a href="https://github.community/t/welcome-to-the-github-actions-board/16022">https://github.community/t5/GitHub-Actions/Welcome-to-the-GitHub-Actions-board/td-p/30298 </a>it is not clear on how long to wait on the waitlist.  Even clarification in terms of days-vs-weeks there would help new users (well, actions are available for my personal account, but not for a new project/organization) to decide on either we should wait or just setup with an alternative CI, at least for now.</p>

**Comment by wipe2238** [comment]
<p>Ye, some edit in linked thread would be nice</p>
<p><a href="https://github.blog/changelog/2019-10-01-auto-admission-in-the-github-actions-and-github-package-registry-beta/" rel="nofollow noopener">https://github.blog/changelog/2019-10-01-auto-admission-in-the-github-actions-and-github-package-registry-beta/</a></p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi <a class="mention" href="/yarikoptic">@yarikoptic</a>! If you join today waitlist today, you’ll have access early next week.</p>

**Comment by yarikoptic** [comment]
<p>Thanks!  so it is days – I can wait (requested yesterday)</p>

**Comment by yarikoptic** [comment]
<p>not sure what that auto-admission is about since I (well – the organization) was not auto-admitted</p>
